lifecycle/lifecycle-viewmodel/src/commonMain/kotlin/androidx/lifecycle/ViewModelProvider.kt

```diff
@@ -99,7 +99,7 @@ public expect class ViewModelProvider {
     }
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    public open class OnRequeryFactory() {
+    public open class OnRequeryFactory {
```

Comment:
Do we still need it in comon?

Reply:
I guess, it doesn't mutter, as it is already commonized, so, no action should be done in the fork